### DISCUSSION TITLE
Adding overlay to video - new setting ignored?

### DISCUSSION THREAD

**Discussion by Gerald-D** [gopro/labs#598] [Q&A]
Using https://gopro.github.io/labs/control/overlays/ for a timestamp overlay on HERO8 to have HH:MM on video

With each change of settings, and scanning QR Code, some of the settings change, but some older settings persist. How do I clear the camera settings to load a fresh one? 

For example, I had "Limit display time" set at 1 second, but want to change it back to unlimited

I have ticked the Erase box in the line: "Make the overlay permanently active: Are you sure? (Risky)  Permanent Overlay  Erase" then the overlay disappears.

Next setup of the QR code, and that 1 second display time is permanent.

I suspect that I checked Permanent in the"Make the overlay permanently active: Are you sure? (Risky)  Permanent Overlay  Erase" line. How do I undo that?

Thank you

**Comment by dnewman-gpsw** [comment]
!RESET - Removes all 

!M[wxzy]=0   - removes a single setting where "wxyz" is the feature you enabled.
e.g. !MBRNT=0

**Reply by Gerald-D** [reply]
Thanks. I am working on a desktop. Where do I create the QR code for !RESET ?

**Reply by dnewman-gpsw** [reply]
https://gopro.github.io/labs/control/custom/

**Reply by Gerald-D** [reply]
I have tried it extensively, but one setting is still persisting.

I want the setting to be: g0!MBRNO=10!MBURN="(200,50)[BRHH:MM]"
But I keep on getting: g0!MBRNT=1!MBRNO=10!MBURN="(200,50)[BRHH:MM]"
Despite having done a !RESET in between

**Reply by dnewman-gpsw** [reply]
If you have existing setting, you have not reset correctly. !RESET requires following the on screen instructions.

**Reply by Gerald-D** [reply]
Thanks for the hints. Found my answer here: https://community.gopro.com/s/question/0D53b00009DLyITCA1/urgent-help-regarding-gp-labs?language=en_US
